server/cmwell-it/src/it/scala/cmwell/it/APIValidationTests.scala

```diff
@@ -159,14 +168,22 @@ class APIValidationTests extends AsyncFunSpec with Matchers with Inspectors with
+        // scalastyle:on
+        executeAfterCompletion(h)(spinCheck(100.millis, true)(Http.get(cmt / "atom1", List("format" -> "atom", "length" -> "10", "op" -> "search"))){
+          r =>
+            val xml = XML.loadString(r.payload)
+            val filtered = filterAllDates(xml)
+            filtered == expected
+        })

```

Comment:
please move the `map` of f27 into this `i`. Also, use only one "expected" and not 2.